proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,393 @@
+
+This proposal puts a possible set of _area teams_ to form with the specified areas of ownership. One notable point is that LLVM (`:/llvm`) had 945 unique authors contribute over the 12 month period used for statistics. This is far more than any other part of the project, and it justified splitting LLVM into two teams, which are separated as the _LLVM Backend_ and _LLVM Middle-end_. Clang had 706 unique authors, which is also significantly higher than other sub-projects. To represent different parts of the Clang community the _Clang_ and _Clang Tooling_ teams are separated below.
+
+* LLVM Backend (CodeGen, MC, Target)
+* LLVM Middle-end (Everything not covered by LLVM Backend)
+* Clang (All the `clang` sub-project except tooling and format libraries and tools)
+* Clang Tooling (`clang-tools-extra` and the tooling and format libraries from `clang`)
+* C/C++ Runtime Libraries (`libcxx`, `libc`, `libunwind`, `libcxxabi`)
+* Compiler Runtimes (`compiler-rt`, `openmp`)
+* Flang (`flang`)
+* MLIR (`mlir`)
+* Binary tools (`lldb`, `lld`, `bolt`)
+* Incubator (`circt`, `torch-mlir`, `Polygeist`, `clangir`, etc)
+* Project Infrastructure (`zorg`, `utils`, `llvm-test-suite`, CI/CD, hosting, etc)
+* Community (Discord, Discourse, etc)
+
+##### Community & Infrastructure Area Teams
+
+The _Community_ and _Infrastructure_ _area teams_ have different responsibilities from the other _area teams_ because they aren’t restricted to specific software components of the LLVM project. Their goals are more holistic to provide for the needs of the whole project and wider community.
+
+Both the _community_ and _infrastructure area teams_ will select one member of their team to act as a liaison with the LLVM Foundation. The liaisons will be responsible for managing the relationship between their _area team_ and the employees and directors of the LLVM Foundation.
+
+##### Voting by Area
+
+Every _voting contributor_ is eligible to vote for the project-wide _Community_ and _Infrastructure_ teams. Eligibility for voting on other _area teams_ is determined by _meaningful contributions_ to the areas those teams cover. For example, in order to vote on the **LLVM Backend** _area team_, a contributor would need to contribute code to the LLVM `CodeGen`, `MC`, or `Target` libraries, or to vote on the members of the **Flang** team, a contributor would need to contribute code to the `flang` project. There is no limit to the number of _area teams_ a contributor can vote on if they have made contributions to the relevant areas.
+
+#### Vacancies
+
+A member of an _area team_ can resign at any time. Additionally as life can sometimes happen unexpectedly a member of an _area team_ may be unable to fulfill their duties or resign. In that case a majority of the remaining _area team_ may vote to declare the member removed in absentia after a 90-day absence.
```

Comment:
Nit: the second sentence can use some commas. I had to reread it twice to understand its meaning.